patterns/sizeTree.ts

```diff
@@ -11,10 +11,19 @@ export function sizeTree<C extends Chain, U, X>(
         Object.values(pallets).map((pallet) => [
           pallet.name,
           Rune.rec(Object.fromEntries(
-            Object.values(pallet.storage).map((entry) => [
-              entry.name,
-              chain.pallet(pallet.name).storage(entry.name).size(undefined!, blockHash),
-            ]) || [],
+            Object.values(pallet.storage).map((entry) => {
+              const partialKeyType: string = entry.partialKey._metadata[0]?.args[2]?._metadata[0]
+                ?.name
+              const partialKey = {
+                $partialEmptyKey: undefined,
```

Comment:
can omit since if you index it and the key doesnt exist it will return` undefined` anyways

Reply:
`null` is a valid partial key for everything, so this lookup isn't needed at all.

Reply:
When changed to null we get the following:

<img width="1008" alt="Screen Shot 2023-03-24 at 12 15 58 PM" src="https://user-images.githubusercontent.com/4893548/227582090-1adcd214-fded-4e7e-bd83-cc5975d11afa.png">

Reply:
Ah, yes – `if (assert.value === null) return` should be added above https://github.com/paritytech/capi/blob/main/frame_metadata/key_codecs.ts#L84